javascript/ql/src/Security/CWE-295/DisablingCertificateValidation.qhelp

```diff
@@ -5,18 +5,69 @@
 
 	<overview>
 
+		<p>
+
+			Certificate validation is the standard authentication method of a
+			secure TLS connection. Without it, there is no guarantee about who the
+			other party of a TLS connection is, enabling man-in-the-middle
```

Comment:
What about: 
`there is no guarantee about who the other party of a TLS connection is, making man-in-the-middle attacks more likely to occur`
instead. It's not because there isn't validation that you necessarily enable MITM attacks, you just make them more likely to happen.